regression/pages/studio/studio_home.py

```diff
@@ -62,3 +62,11 @@ def click_privacy_policy(self):
         """
         self.q(
             css='a[href="' + BASE_URL_LMS + '/edx-privacy-policy"]').click()
+
+    def click_course_rerun(self):
+        """
+        Clicks rerun course button
+        """
+        self.browser.execute_script(
```

Comment:
@benpatterson 
Button appears after hovering on it, I tried using move_to_element but the tests were flaky. So used JS to make it more reliable.